astropy_helpers/commands/build_ext.py

```diff
@@ -303,7 +304,7 @@ def run(self):
             # Update cython_version.py if building with Cython
             try:
                 cython_version = get_pkg_version_module(
-                        packagename, fromlist=['cython_version'])[0]
+                    packagename, fromlist=['cython_version'])[0]
```

Comment:
Same here, this would probably be ok as a single line

Reply:
the file has lines very much around 80, this would be 100, so I'll as is.